tests/test_libworkflow.py

```diff
@@ -19,13 +20,14 @@ def test_WorkflowManager(caplog):
             }
         }
     with tempfile.TemporaryDirectory(dir=".") as _tmpdir:
-        workflow = WorkflowManager(
-            ParamDict,
-            start=0,
-            other_params=Any,
-            )
-        workflow.postprocess()
-        first_log_line = str(caplog.records[0].message)
-        second_log_line = str(caplog.records[1].message)
-        assert first_log_line == "Reading instructions step 0_topoaa"
-        assert second_log_line == "Running haddock3-analyse on ./, modules [], with top_cluster = 10"  # noqa : E501
+        with working_directory(_tmpdir):
```

Comment:
Why do you need to change the context here?

Reply:
otherwise the temporary directory is simply not used.